main.py

```diff
@@ -105,14 +105,14 @@ def print_standings(league_table):
 					position="POS", team="CLUB", played="PLAYED", 
 					goaldiff="GOAL DIFF", points="POINTS"
 			))
-
+	positionlist = (team["position"] for team in league_table["standing"])
```

Comment:
I am getting a `TypeError: object of type 'generator' has no len()`. Changing the generator comprehension to a list comprehension seemed to fix this for me. And it does print the bottom 3 in red on 18 and 20 team leagues which is an improvement.